<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWdg</class>
 <widget class="QWidget" name="RoomWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视频会议主界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdgTop" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="titleBar">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>23</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="lblLogo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>242</width>
            <height>46</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRoomNameLogo">
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/skin/room/icon_rom_name.png);
background-repeat: no-repeat;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRoomName">
          <property name="styleSheet">
           <string notr="true">border: 0px;
font-size:12pt;
font-family:微软雅黑,宋体;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="sysBtnBar">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdgClient" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>2</number>
        </property>
        <property name="childrenCollapsible">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="leftFrm" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(245, 245, 245);</string>
         </property>
        </widget>
        <widget class="QWidget" name="centerFrm" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(245, 245, 245);</string>
         </property>
         <layout class="QHBoxLayout" name="layoutRight" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="btnTurnLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>5</width>
              <height>9</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>5</width>
              <height>9</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>5</width>
              <height>9</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="videoFrm" native="true">
            <property name="maximumSize">
             <size>
              <width>6000</width>
              <height>6000</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdgBotton" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="bottomToolBar">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="btnMic">
          <property name="toolTip">
           <string>静音</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox:focus, QRadioButton:focus 
{
	border: none;
	outline: none;
}

QCheckBox
{
	font-size:12px;
	font-family:微软雅黑,宋体;
}

QCheckBox::indicator
{	
	width:22px;
	height:22px;
}
QCheckBox::indicator:unchecked 
{
	border-image: url(:/skin/room/mic_check.png) 0 110 0 0;
}

QCheckBox::indicator:unchecked:hover
{
	border-image: url(:/skin/room/mic_check.png) 0 88 0 22;
}

QCheckBox::indicator:unchecked:pressed
{
	border-image: url(:/skin/room/mic_check.png) 0 66 0 44;
}

QCheckBox::indicator:unchecked:disabled
{
  border-image: url(:/skin/room/mic_check.png) 0 66 0 44;
}

QCheckBox::indicator:checked
{
	border-image: url(:/skin/room/mic_check.png) 0 44 0 66;
}

QCheckBox::indicator:checked:hover
{
	border-image: url(:/skin/room/mic_check.png) 0 22 0 88;
}

QCheckBox::indicator:checked:pressed
{
	border-image: url(:/skin/room/mic_check.png) 0 0 0 110;
}

QCheckBox::indicator:checked:disabled
{
    border-image: url(:/skin/room/mic_check.png) 0 0 0 110;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderMic">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="btnSpeaker">
          <property name="toolTip">
           <string>静音</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox:focus, QRadioButton:focus 
{
	border: none;
	outline: none;
}

QCheckBox
{
	font-size:12px;
	font-family:微软雅黑,宋体;
}

QCheckBox::indicator
{	
	width:22px;
	height:22px;
}
QCheckBox::indicator:unchecked 
{
	border-image: url(:/skin/room/speaker_check.png) 0 110 0 0;
}

QCheckBox::indicator:unchecked:hover
{
	border-image: url(:/skin/room/speaker_check.png) 0 88 0 22;
}

QCheckBox::indicator:unchecked:pressed
{
	border-image: url(:/skin/room/speaker_check.png) 0 66 0 44;
}

QCheckBox::indicator:unchecked:disabled
{
  border-image: url(:/skin/room/speaker_check.png) 0 66 0 44;
}

QCheckBox::indicator:checked
{
	border-image: url(:/skin/room/speaker_check.png) 0 44 0 66;
}

QCheckBox::indicator:checked:hover
{
	border-image: url(:/skin/room/speaker_check.png) 0 22 0 88;
}

QCheckBox::indicator:checked:pressed
{
	border-image: url(:/skin/room/speaker_check.png) 0 0 0 110;
}

QCheckBox::indicator:checked:disabled
{
    border-image: url(:/skin/room/speaker_check.png) 0 0 0 110;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderSpeaker">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFullScreen">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>全屏显示视频或文档</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resources/Default.qrc">
            <normaloff>:/video/fullscreen_24_2.png</normaloff>:/video/fullscreen_24_2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnShowPPT">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip">
           <string>演示PPT</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resources/Default.qrc">
            <normaloff>:/skin/room/ppt_1h.png</normaloff>:/skin/room/ppt_1h.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>67</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottom_right">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnShowVideo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>99</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>显示我的视频</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloseVideo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>99</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>显示我的视频</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Default.qrc"/>
 </resources>
 <connections/>
</ui>
